Applications/WebPortal/WebApp/library/source/pages/account_directory_page/account_directory_model.ts

```diff
@@ -0,0 +1,17 @@
+import * as Beam from 'beam';
+import { AccountEntry } from './account_entry';
+
+/** Interface for a model representing an directory of accounts. */
+export abstract class AccountDirectoryModel {
+
+  /** Loads this model. */
+  public abstract async load(): Promise<void>;
+
+  /** Returns a list of all groups. */
+  public abstract get groups(): Beam.Set<Beam.DirectoryEntry>;
```

Comment:
Accessors come before functionality, so put the getters/setters before the ```load``` method. ```loadAccounts``` is considered an accessor whereas ```load``` is considered functionality owing to the fact that it actually mutates the state of the model.